res/photon/index.html

```diff
@@ -225,6 +225,15 @@ <h3 class='photon-title-20'>Photon Message Bar Generic</h3>
             aria-label="Hide the message" title="Hide the message">
           </button>
         </div>
+        <div class="photon-message-bar multi-line">
```

Comment:
We shouldn't add a new class name for this. We can do it by wrapping this `<div class="photon-message-bar">` with a `<div style="width: 300px;">` instead. We should do this since we don't want to add a multi-line class on our other components as well.